### DISCUSSION TITLE
Is there a more comprehensible overview of all the different meshes?

### DISCUSSION THREAD

**Discussion by njiels** [Tresjs/tres#348] [Ideas]
I've just started with tresjs, it's nice but imo a bit vague in it's explenation of the different mesh elements and it's arguments. Is there a cheat sheet somewhere or could that be uploaded to the documentation?

**Comment by OverRevvv** [comment]
### Yes, pretty much every detail related to Tresjs is listed in the documentation, if you want to explore more about Tresjs or having problems with it, I'd also recommend joining the discord server where you can discuss everything related to Tresjs, the server is excellent and helpful.

###  Here is the link [Discord Tresjs community ]( https://discord.com/invite/UCr96AQmWn)

### Check out the documentation as well, it's also fantastic and helpful.

### [https://tresjs.org/examples/load-models.html](https://tresjs.org/examples/load-models.html)
### [https://cientos.tresjs.org/guide/abstractions/text-3d.html](https://cientos.tresjs.org/guide/abstractions/text-3d.html)

### Also, you should check this playlist on Youtube which was made by @alvarosabu  the creator of Tresjs.

### [Tresjs Playlist](https://www.youtube.com/playlist?list=PLi-X1Ojrrmi_V15dXJf_XKdss0dlDzSwa)

##  Have a great time with Tresjs!

**Comment by alvarosabu** [comment]
Hey @njiels all the different meshes, materials, geometries, etc are the same as ThreeJS https://threejs.org/docs/index.html there is no need to duplicate documentation.

You just need to add the `prefix` Tres to the components. Arguments are the same as the constructor's parameters. https://tresjs.org/api/instances-arguments-and-props.html

Cheers
